PhysicsTools/PatUtils/plugins/L1ECALPrefiringWeightProducer.cc

```diff
@@ -214,12 +214,12 @@ L1ECALPrefiringWeightProducer::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<edm::InputTag>("TheJets", edm::InputTag("slimmedJets"));
   desc.add<std::string>("L1Maps","L1PrefiringMaps.root");
   desc.add<std::string>("DataEra", "2017BtoF");
-  desc.add<bool>("UseJetEMPt", true);
+  desc.add<bool>("UseJetEMPt", false);
   desc.add<double>("PrefiringRateSystematicUncty",0.2);
-  desc.add<bool>("SkipWarnings", true);
-  
-  descriptions.add("l1ECALPrefiringWeightProducer",desc);
+  desc.add<bool>("SkipWarnings", false);
 
+  descriptions.add("prefiringweight", desc);
```

Comment:
It is safer to have a default config with a generic name, as you had before ("l1ECALPrefiringWeightProducer") and clone and modify the specialization ("prefiringweight") in the cff file (PhysicsTools/PatUtils/python/L1ECALPrefiringWeightProducer_cff.py), or whatever other config users want to use for it.